nextflow.config

```
@@ -47,8 +47,9 @@ params {
     annotate_mutations         = false
     perform_qcs                = true
     split_by_chrom             = false
+    run_splitfastq             = false
     splitfastq_parts           = 20
-    splitted_original_sample   = false
+    splitted_original_sample   = true
```

we should change this to true in the test.config, not in the main nextflow.config
```suggestion
    splitted_original_sample   = false
```